### PR TITLE
fix(ui): Minor UI fixes and enhancements + more keyboard shortcuts

### PR DESCRIPTION
**Pull request by rbreeze**
Adds a message when no rollouts exist in namespace:
Screen Shot 2021-04-13 at 10 08 43 AM

Adds a help menu for keyboard shortcuts:
Screen Shot 2021-04-13 at 10 09 29 AM

Signed-off-by: Remington Breeze <remington@breeze.software>

Checklist:

* [ ] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-rollouts/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this is a chore.
* [ ] The title of the PR is (a) [conventional](https://www.conventionalcommits.org/en/v1.0.0/), (b) states what changed, and (c) suffixes the related issues number. E.g. `"fix(controller): Updates such and such. Fixes #1234"`.  
* [ ] I've signed my commits with [DCO](https://github.com/argoproj/argoproj)
* [ ] I have written unit and/or e2e tests for my change. PRs without these are unlikely to be merged.
* [ ] My builds are green. Try syncing with master if they are not. 
* [ ] My organization is added to [USERS.md](https://github.com/argoproj/argo-rollouts/blob/master/USERS.md).

### CODE REVIEW

File: ui/src/app/components/pods/pods.scss [hunk 1]
```diff
@@ -15,7 +15,6 @@ $POD_SIZE: 30px;
     }
     &__container {
         margin-top: 10px;
-        // border: 1px solid $argo-color-gray-5;
```
Comment (alexmt) [inline ui/src/app/components/pods/pods.scss:18]: Do you want to remove commented code?

**Review by alexmt — APPROVED** [review]
LGTM

File: ui/src/app/components/input/input.scss [hunk 2]
```diff
@@ -3,26 +3,43 @@
...
     box-sizing: border-box;
+    border-radius: 5px;
+    padding: 7px 10px;
+    background-color: $argo-color-gray-3;
+    border: 1px solid $argo-color-gray-5;
+    color: $argo-color-gray-7;
+    display: flex;
+    align-items: center;
+    overflow: hidden;
+    padding: 12px;
```
Comment (alexmt) [inline ui/src/app/components/input/input.scss:14]: padding is specified twice.

File: ui/src/app/components/pods/pods.scss (hunk 1 above)
Comment (rbreeze) [reply ui/src/app/components/pods/pods.scss:18]: Yeah, this was from old styles that I decided to remove

**Pull request merged**